op-service/ctxinterrupt/context_test.go

```diff
@@ -0,0 +1,19 @@
+package ctxinterrupt
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestContextKeyIsUnique(t *testing.T) {
```

Comment:
This is probably overkill, but I never realised wrapper types of struct{} avoided equality with each other. Probably worth removing if this is obvious. In earlier versions of Go, struct{} comparison was janky.

Reply:
It's a bit overkill yes. Typing is strong, and thus different. But I'm fine keeping this test around